tests/bench_ztimer/main.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       ztimer set / remove / now benchmark application
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "test_utils/expect.h"
+
+#include "msg.h"
+#include "thread.h"
+#include "ztimer.h"
+
+#ifndef ZTIMER
+#define ZTIMER ZTIMER_MSEC
+#endif
+
+#ifndef NUMOF_TIMERS
+#define NUMOF_TIMERS   (1000U)
+#endif
+
+#ifndef REPEAT
+#define REPEAT   (1000U)
+#endif
+
+#ifndef BASE
+#define BASE    (100000LU)
```

Comment:
```suggestion
#define BASE    (1000000LU)
```
Without this with `frdm-kw41z` I actually have it triggering early...

Reply:
I've increased to 10000000 (10 million). It just has to be far enough in the future to never trigger.